scripts/ktlint_lint_check.sh

```diff
@@ -17,7 +17,7 @@ fi
 committed_files=`git diff --name-only HEAD^ HEAD -- '***.kt'`
 
 if [ -z "$committed_files" ]; then
-    echo No kotlin files have been committed, skipping ktlinter.
+    echo "No kotlin files have been committed, skipping ktlinter."
```

Comment:
"skipping file formatting checks on kotlin file" how this sounds?

Reply:
Sounds more informative, I will update a commit. 
Also, for the situation when a user runs the `ktlint_lint_check.sh` before committing a file, no checks will be performed (as these changes are not committed). Should we also add a statement saying "Please commit the files that you want to check..." or something like that, which could prompt the user to commit and then run the script again ?

Reply:
> "skipping file formatting checks on kotlin file" how this sounds?

Done.